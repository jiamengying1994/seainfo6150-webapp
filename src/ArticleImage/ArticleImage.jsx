import React from "react";
import styles from './ArticleImage.module.css';

export class ArticleImage extends React.Component {
  constructor(props) {
      super(props);

   }

  render() {

  return (
    <div className={styles.article_image}>
    <img src={this.props.url} alt={this.props.title}></img>
    </div>
    );
  }
}

export default ArticleImage;
