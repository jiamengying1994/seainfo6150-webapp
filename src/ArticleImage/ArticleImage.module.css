img
{
  height: 80%;
  width: 60%;
  position: relative;
  right: auto;
  border: 5px solid red;
  border-radius: 100px/80px;
  box-shadow: 5px 5px 5px gray;
	background: red;
}

.article_image
{

	width: 100%;
	height: auto;
	overflow: hidden;
	position: relative;
    float: left;
}