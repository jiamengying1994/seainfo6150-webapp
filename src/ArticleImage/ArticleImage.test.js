import React from "react";
import ArticleImage from "./ArticleImage";
import { fireEvent } from "@testing-library/react";

describe("ArticleImage test", () => {


  it("renders correctly", () => {
    const al = {"url": "https://cdn.cnn.com/cnnnext/dam/assets/181122102925-statue-of-liberty-crown-super-169.jpg"}
    const { container} = render(
      <ArticleImage articleImage={al}/>
    );

    expect(container).toMatchSnapshot();
  });
});