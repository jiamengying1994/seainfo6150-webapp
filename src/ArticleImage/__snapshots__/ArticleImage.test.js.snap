// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleImage test renders correctly 1`] = `
<div>
  <div
    class="article_image"
  >
    <img />
  </div>
</div>
`;
